instrument/HRPD_Definition_pre20210301.xml

```diff
@@ -0,0 +1,864 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!-- For help on the notation used to specify an Instrument Definition File 
+     see http://www.mantidproject.org/IDF -->
+<instrument xmlns="http://www.mantidproject.org/IDF/1.0" 
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xsi:schemaLocation="http://www.mantidproject.org/IDF/1.0 http://schema.mantidproject.org/IDF/1.0/IDFSchema.xsd"
+ name="HRPD" valid-from   ="2007-01-01 23:59:59"
```

Comment:
The old valid-from date was 1900 i.e. included all historic runs. Is there a reason it's now 2007? What would be used for runs older than this, if there are any?

Reply:
Yep good catch sorry - don't know where 2007 came from...perhaps the file I was sent? I will change to 1900 to be safe.